conan_support.cmake

```diff
@@ -163,10 +166,78 @@ function(conan_install)
 endfunction()
 
 
+function(conan_version_parse result conan_version conan_version_raw)
```

Comment:
I think I would make it an internal implementation detail of ``conan_get_current_version``, to avoid cluttering ``conan_support.cmake`` with many public-like functions

Reply:
In any case, the purpose of this function is not fully clear. CMake knows how to compare versions as strings, no need for manual parsing.

Reply:
Yeah, it doesn't really serve a purpose now that the code uses CMake's version comparison so it's now deleted.